src/i18n/sv.js

```diff
@@ -0,0 +1,102 @@
+      "password_leaked": "Din konto har blivit spärrat då ditt lösenord har lagts upp på en annan webbsida. Du har fått e-post med instruktioner om hur du låser upp ditt konto.",
+      "too_many_attempts": "Ditt konto har blivit spärrat efter flera felaktiga inloggningsförsök."
+    },
+    passwordless: {
+      "bad.email": "Din e-postadress är ogiltig.",
+      "bad.phone_number": "Ditt telefonnummer är ogiltigt.",
+      "lock.fallback": "Något gick fel."
+    },
+    signUp: {
+      "invalid_password": "Lösenordet är ogiltigt.",
+      "lock.fallback": "Något gick fel vid registreringen.",
+      "password_dictionary_error": "Lösenordet är för vanligt.",
+      "password_no_user_info_error": "Lösenordet baseras på personuppgifter.",
+      "password_strength_error": "Lösenordet är för svagt.",
+      "user_exists": "Användaren finns redan.",
+      "username_exists": "Användarnamnet finns redan."
+    }
+  },
+  success: { // success messages show above the form or in a confirmation pane
+    logIn: "Tack för att du loggar in.",
+    forgotPassword: "Vi har skickat dig ett e-postmeddelande där du kan återställa ditt lösenord.",
+    magicLink: "Vi har skickat en länk för att logga in<br />till %s.",
+    signUp: "Tack för att du registrerade dig."
+  },
+  blankErrorHint: "Får inte lämnas tom",
+  codeInputPlaceholder: "Din kod",
+  databaseEnterpriseLoginInstructions: "",
+  databaseEnterpriseAlternativeLoginInstructions: "eller",
+  databaseSignUpInstructions: "",
+  databaseAlternativeSignUpInstructions: "eller",
+  emailInputPlaceholder: "dinepostadress@exempel.se",
+  enterpriseLoginIntructions: "Logga in med dina företagsuppgifter.",
+  enterpriseActiveLoginInstructions: "Ange dina företagsuppgifter för %s.",
+  failedLabel: "Misslyckades!",
+  forgotPasswordAction: "Glömt ditt lösenord?",
+  forgotPasswordInstructions: "Ange din e-postadress. Vi skickar ett e-postmeddelande med instruktioner för att återställa ditt lösenord.",
+  forgotPasswordSubmitLabel: "Skicka e-post",
+  invalidErrorHint: "Ogiltig",
+  lastLoginInstructions: "Senast loggade du in som",
+  loginAtLabel: "Logga in på %s",
+  loginLabel: "Logga in",
+  loginSubmitLabel: "Logga in",
+  loginWithLabel: "Logga in med %s",
+  notYourAccountAction: "Inte ditt konto?",
+  passwordInputPlaceholder: "ditt lösenord",
+  passwordStrength: {
+    containsAtLeast: "Måste innehålla minst %d av följande %d sorters tecken:",
+    identicalChars: "Inte fler än %d identiska tecken i rad (t.ex., \"%s\" är inte tillåtet)",
+    nonEmpty: "Lösenord får inte vara tomma",
+    numbers: "Nummer (t.ex. 0-9)",
+    lengthAtLeast: "Minst %d tecken långt",
+    lowerCase: "Gemener (a-z)",
+    shouldContain: "Måste innehålla:",
+    specialCharacters: "Specialtecken (t.ex. !@#$%^&*)",
+    upperCase: "Versaler (A-Z)"
+  },
+  passwordlessEmailAlternativeInstructions: "Annars kan du ange din e-postadress för att logga in<br>eller skapa ett konto",
```

Comment:
Hmm, this has gone from an instruction to an information about a possibility (like "Otherwise you could…").  It's a bit hard to translate without seeing what precedes this, but I think "Annars, ange din e-postadress …" without "kan".